tests/functional/cylc-subscribe/01-subscribe.t

```diff
@@ -38,21 +38,12 @@ run_ok "${TEST_NAME}" cylc subscribe --once --topics="workflow" "${SUITE_NAME}"
 grep_ok "running" "${TEST_NAME}.stdout"
 
 TEST_NAME="${TEST_NAME_BASE}-subscribe-2"
-# Need fields from "cylc scan", cannot quote
-# shellcheck disable=SC2046
-run_ok "${TEST_NAME}" cylc subscribe --once --topics="workflow" \
-    $(cylc scan --color=never --publisher -n "${SUITE_NAME}" \
-        | awk -F' ' -v RS='\n([ \t]*\n)+' '{print $1 " " $3}')
```

Comment:
Didn't need to provide host:port via `cylc scan`, the internal logic does this for us.